lib/github/templates/security-pre-release.md

```diff
@@ -0,0 +1,35 @@
+---
+date: %ANNOUNCEMENT_DATE%
+category: vulnerability
+title:  %RELEASE_DATE% Security Releases
+slug: %SLUG%
+layout: blog-post.hbs
+author: %AUTHOR%
+---
+
+# Summary
+
+The Node.js project will release new versions of the %AFFECTED_VERSIONS%
+releases lines on or shortly after, %RELEASE_DATE% in order to address:
+
+%VULNERABILITIES%
+
+## OpenSSL Security updates
+
+This security release includes the following OpenSSL security updates
+
+%OPENSSL_UPDATES%
+
```

Comment:
Not every security release will contain OpenSSL updates.

Reply:
Yes that should be optional, Im still trying to figure out how to automate the check of there is an openssl update

Reply:
I'll leave only the placeholder for now